source/contracts/reporting/Reporting.sol

```diff
@@ -14,7 +14,8 @@ library Reporting {
     uint256 private constant LIMITED_REPORTERS_DISPUTE_BOND_AMOUNT = 11 * 10**21;
     uint256 private constant ALL_REPORTERS_DISPUTE_BOND_AMOUNT = 11 * 10**22;
 
-    uint256 private constant GAS_TO_REPORT = 567796;
+    // NOTE: We need to maintain this cost to roughly match the gas cost of reporting. This was last updated 10/02/2017
+    uint256 private constant GAS_TO_REPORT = 600000;
```

Comment:
Did you actually run this number?  Is it for real?  😢

Reply:
Tis real. The target reporter gas cost fee is quite large due to this as you can imagine. :[

Reply:
If you look at the test I had in place before btw that is how I was calculating it